test/ziggurat/messaging/producer_test.clj

```diff
@@ -333,7 +333,7 @@
             (is (= 0 @counter)))))
 
       (testing "it creates queues with topic entity for channels only"
-        (with-open [ch (lch/open rmqw/connection)]
+        (with-open [ch (lch/open @rmqw/connection)]
```

Comment:
why not define a fn that returns the connection state, instead of derefing it here?